packages/zoe/NEWS.md

```diff
@@ -1,4 +1,90 @@
 User-visible changes in @agoric/zoe:
+
+## Release 0.3.0 (?)
+
+The structure of offer rules has changed to:
+
+```js
+{
+  offer: { Asset: moola(4 )},
+  want: { Price: simoleans(15) },
+  exit: { afterDeadline: {
+    timer,
+    deadline: 100,
+  }}
+}
+```
+There are no longer any keys such as 'payoutRules' or 'exitRule'. Most
+importantly, we no longer rely on the specific order of payoutRules
+arrays and payment/payout arrays to be the same. In fact, we can even
+make partial offerRules that will be filled in. 
+
+`Asset` and `Price` in the above example are called "role names". Each
```

Comment:
This is the place to point out that the role names describe the purposes and distinct uses of issuers in the contract, and don't have anything to do with the different seats, which control the actions that participants in a contract can take. Once we're talking about that, we also need to bring up the duplication issues, and all that.

Reply:
Do you think we still need to explain any of that with the renaming. If we just say "contracts have keywords" I think readers will accept it. I think that's the wrong place to bring up duplication issues.